testing/resources/toml_bad.toml

```diff
@@ -0,0 +1,3 @@
+key = # INVALID
+
+= "no key name"  # INVALID
```

Comment:
instead of having invalid toml files checked in, can you write them as temporary files in the tests -- I realize there's old hooks which do the former but I've been preferring the latter for everything new -- thanks!